framework/src/task_manager/task_manager_set_callback.c

```diff
@@ -254,10 +280,31 @@ int task_manager_set_stop_cb(void (*func)(void *data), void *cb_data)
 	if (request_msg.data == NULL) {
 		return TM_OUT_OF_MEMORY;
 	}
-	((tm_termination_info_t *)request_msg.data)->cb = (_tm_termination_t)func;
-	((tm_termination_info_t *)request_msg.data)->cb_data = cb_data;
+	REQ_CB = (_tm_termination_t)func;
```

Comment:
I don't know what it is exactly unless I find define macro.
I think that it is better to keep request_msg in define at least.
For example, REQ_CB(request_msg) = (_tm_termination_t)func;